.circleci/config.yml

```diff
@@ -50,7 +52,7 @@ jobs:
               git checkout gh-pages || git checkout --orphan gh-pages
               git rm -rf .
               cd ..
-              ~/miniconda/envs/mibitracker-client-dev/bin/sphinx-apidoc -f -o docs/source ./mibitracker ./mibitracker/{**,}/tests
+              ~/miniconda/envs/mibitracker-client-dev/bin/sphinx-apidoc -f -o docs/source ./ */tests ./setup.py
```

Comment:
Because the previous one worked for me on my osx laptop (and didn't have the issue you saw on your machine), I checked this version using the circleci local cli and it worked fine.